dev/integration_tests/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -198,11 +209,16 @@ class _ChipDemoState extends State<ChipDemo> {
 
   @override
   Widget build(BuildContext context) {
-    final List<Widget> chips = _materials.map<Widget>((String name) {
+    final ThemeData theme = Theme.of(context);
+    final List<Widget> chips = _materialsA.map<Widget>((String name) {
       return Chip(
         key: ValueKey<String>(name),
-        backgroundColor: _nameToColor(name),
-        label: Text(_capitalize(name)),
+        backgroundColor: theme.colorScheme.secondaryVariant,
```

Comment:
This seems to fundamentally change what the demo had been doing (generating a range of background colors). Seems OK to me and I assume that it restores sufficient contrast for a11y.

Reply:
Yeah, this is where I spent most of my time trying to figure out how to keep it. It was with the addition of the dark theme that this failed. Since the text color changed (to a light text color), the chip color (based on string hash) contrast accessible. Maybe I should instead change the text to always be dark? 
I thought maybe basing color on hash value wasn't reliably accessible in general. 

@gspencergoog do you have any ideas on how to maintain the color-from-name in an accessible way? I think you authored it originally.

Reply:
I think that if you were to change the "value" of the HSV from a fixed 0.9 (fairly light) to the "value" of the color scheme's background color that you'd probably get something accessibility friendly. The hash only changes the hue of the color, not the value.  That said, HSV isn't totally perceptually accurate, so it might be possible to get a non-accessible combination.

The goal of the code was to generate unique colors for each chip, based on the name.

Reply:
You could probably also use the value of the color scheme's primary color.

Reply:
Ah! Thank you! I will give that a try!

Reply:
I was able to restore this feature with your suggestion, thanks @gspencergoog!